bahmni-docker/docker-compose.yml

```diff
@@ -2,8 +2,11 @@ version: "3.7"
 services:
   proxy:
     image: "bahmni/proxy:${PROXY_IMAGE_TAG:?}"
+    volumes: 
+      - ../bahmni-proxy/resources/tls:/etc/tls
```

Comment:
I feel that it would be nice to have the volume mounted folder inside `bahmni-docker` directory itself (Ex: `./ssl-certificates`) so that the implementer can easily update the certificate files.